vsts-fullbuild-redhat-vm/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vstsAccountURL": {
```

Comment:
remove params that are required, otherwise deployment will go with empty strings